e2e-tests/portal/cypress/e2e/journeys/maker-bond/bond.spec.js

```diff
@@ -23,7 +23,7 @@ context('Add a Bond to a Deal', () => {
 
   beforeEach(() => {
     cy.deleteDeals(ADMIN);
-    cy.insertOneDeal(MOCK_DEAL, BANK1_MAKER1).then((insertedDeal) => {
+    cy.createBssDeal({ readyForCheck: true }).then((insertedDeal) => {
```

Comment:
I think this should be `cy.createBssDeal({});` so that the deal has a `DRAFT` status